ktor-utils/common/src/io/ktor/util/StringValues.kt

```diff
@@ -92,9 +93,9 @@ open class StringValuesSingleImpl(
 
     override fun names(): Set<String> = setOf(name)
 
-    override fun toString() = "StringValues(case=${!caseInsensitiveName}) ${entries()}"
+    override fun toString(): String = "StringValues(case=${!caseInsensitiveName}) ${entries()}"
```

Comment:
This change doesn't relate to the original issue

Reply:
Could you clarify?